docs/debugging.md

```diff
@@ -1,7 +1,16 @@
 # Debug onos-config in Onit Using Delve
 
-## Debugging Example #1
-- In this tutorial, we assume you already started an onit cluster. If you haven't done that yet, refer to [testing](testing.md) for more details. 
+## Debugging Example #1: Use Goland IDE (Recommended):
+- In this example, we assume you already started an onit cluster. If you haven't done that yet, refer to [testing](testing.md) for more details.
+
+- First, you need to configure your [Goland](https://www.jetbrains.com/go/) IDE to connect to the remote Delve debugger which is running in the onos-config 
+pod. You can do that by going to *Run | Edit Configurations… | + | Go Remote* and configuring the host (in this example *localhost*) and port (in this example port **40000**) your remote debugger is listening on.
```

Comment:
is this port fixed or does it change every time ? if so is there a way to get it ?

Reply:
This is the port that is used in the debug image to start the dlv server and it is fixed.